assets/sass/v2/_number-challenge.scss

```diff
@@ -0,0 +1,33 @@
+.number-challenge-section {
```

Comment:
Shall we scope all v2 css under the v2 specific top level class name?
Not specific to `number-challenge.scss` but also some other files.

Reply:
Are you suggesting we have some prefix that we add to top level class names ? Something like `oie-number-challenge-section` and so on for others ?

We already have all the sass inside a v2 folder so maybe it might be an overkill. Another thing to think about is are we breaking any CSS customizations a customer may have by adding new classnames.

Reply:
Check `_device-challenge.scss`, it scopes css under `.siw-main-view`.
Such pattern probably works all all other css that meant for the widget page body.
(could do as separated refactoring PR)

Reply:
Ah makes sense to scope it under .siw-main-view